samples/ClientSample/HubSample.cs

```diff
@@ -26,44 +24,40 @@ internal class HubSample
             loggerFactory.AddConsole(LogLevel.Debug);
             var logger = loggerFactory.CreateLogger<Program>();
 
-            using (var httpClient = new HttpClient(new LoggingMessageHandler(loggerFactory, new HttpClientHandler())))
+            logger.LogInformation("Connecting to {0}", baseUrl);
+            var connection = new HubConnection(new Uri(baseUrl), new JsonNetInvocationAdapter(), loggerFactory);
+            try
             {
-                logger.LogInformation("Connecting to {0}", baseUrl);
-                var transport = new LongPollingTransport(httpClient, loggerFactory);
-                var connection = new HubConnection(new Uri(baseUrl), new JsonNetInvocationAdapter(), loggerFactory);
-                try
-                {
-                    await connection.StartAsync(transport, httpClient);
-                    logger.LogInformation("Connected to {0}", baseUrl);
-
-                    var cts = new CancellationTokenSource();
-                    Console.CancelKeyPress += (sender, a) =>
-                    {
-                        a.Cancel = true;
-                        logger.LogInformation("Stopping loops...");
-                        cts.Cancel();
-                    };
+                await connection.StartAsync();
```

Comment:
Do we want this to still use LongPolling here? the TransportFactory uses WebSockets by default? 
I also think it would be good to show usage of the enum in the sample.